recipes/libboxes/all/conanfile.py

```diff
@@ -0,0 +1,122 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.env import VirtualBuildEnv
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, replace_in_file, rm, rmdir
```

Comment:
```suggestion
from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, rm, rmdir
```

Please, keep only imports that are used.

Reply:
Will remove.  Thanks!